### PR TITLE
[O2B-1355] API for fetching synchronous QC flags

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- NA

Notable changes for developers:
- Added API for fetching synchronous QC flags

Changes made to the database:
- NA

### CODE REVIEW

File: test/api/qcFlags.test.js [hunk 1]
```diff
@@ -217,6 +217,35 @@ module.exports = () => {
         });
     });
 
+    describe('GET /api/qcFlags/synchronous', () => {
+        it('should successfully fetch synchronous flags', async () => {
```
Comment (martinboulais) [inline test/api/qcFlags.test.js:221]: You should put more specific names, here you have 2 tests with the same name

File: lib/database/seeders/20240404100811-qc-flags.js [hunk 2]
```diff
@@ -126,6 +126,84 @@ module.exports = {
...
+
+                        flag_type_id: 3, // Good
+                        run_number: 56,
+                        created_by_id: 2,
+                        detector_id: 7, // FT0
+
+                        created_at: '2024-08-12 12:00:00',
+                        updated_at: '2024-08-12 12:00:00',
+                    },
+                    {
```
Comment (martinboulais) [inline lib/database/seeders/20240404100811-qc-flags.js:147]: Do we need so much data in seeders?

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/database/seeders/20240404100811-qc-flags.js (hunk 2 above)
Comment (xsalonx) [reply lib/database/seeders/20240404100811-qc-flags.js:147]: yes

File: test/api/qcFlags.test.js (hunk 1 above)
Comment (xsalonx) [reply test/api/qcFlags.test.js:221]: done

**Review by martinboulais — APPROVED** [review]

**Pull request merged**